pkgs/dartpad_ui/lib/simple_widgets.dart

```diff
@@ -515,6 +316,9 @@ class GeneratingCodeDialog extends StatefulWidget {
   State<GeneratingCodeDialog> createState() => _GeneratingCodeDialogState();
 }
 
+bool get isNonMac => defaultTargetPlatform != TargetPlatform.macOS;
```

Comment:
If these are going to be public now (other libraries will need to call?), perhaps move to someplace like `utils.dart`. Or, define at or near the top of this file, so that they're not lost in with the larger widgets.